packages/lib/src/vis/shared/PanZoomMesh.tsx

```diff
@@ -52,13 +58,20 @@ function PanZoomMesh() {
     [abscissaScale, ordinateScale, camera, visWidth, visHeight, invalidate]
   );
 
-  const onPointerDown = useCallback((evt: ThreeEvent<PointerEvent>) => {
-    const { sourceEvent, unprojectedPoint } = evt;
-    const { target, pointerId } = sourceEvent;
-    (target as Element).setPointerCapture(pointerId); // https://stackoverflow.com/q/28900077/758806
+  const onPointerDown = useCallback(
+    (evt: ThreeEvent<PointerEvent>) => {
+      const { sourceEvent, unprojectedPoint } = evt;
+      const { target, pointerId } = sourceEvent;
+      (target as Element).setPointerCapture(pointerId); // https://stackoverflow.com/q/28900077/758806
```

Comment:
Doesn't capturing the pointer here interfere with the mesh selection (or breaks the pointer capture in the selection mesh)? Might be worth moving this down after the event validity check.

Reply:
It seems to work fine in the story.

This being said, I think the pointer capture must be revisited: I noticed some hiccups when releasing the pointer outside the canvas.

Reply:
Still, it makes sense to capture the pointer only if the rest of the callback is executed, you are right.